app/models/preserved_object.rb

```diff
@@ -9,4 +9,21 @@ class PreservedObject < ApplicationRecord
   validates :druid, presence: true, uniqueness: true, format: { with: DruidTools::Druid.pattern }
   validates :current_version, presence: true, numericality: { only_integer: true, greater_than: 0 }
   validates :preservation_policy, null: false
+
+  def self.normalize_druid_name
+    targets = PreservedObject.where("druid LIKE 'druid:%'")
```

Comment:
Is this sanitized enough for our purposes? Maybe it doesn't matter, since this isn't exposed to the world, but:

```ruby
PreservedObject.where("druid LIKE ?", "druid:%")
```

feels safer. Not a blocker.

Reply:
i _think_ we use that syntax (`PreservedObject.where("druid LIKE ?", "druid:%")
)` if we are going to interpolate a variable and are worried about a sql injection, but since we are just hard coding `'druid:%'` it doesn't matter...?

@jmartin-sul  is that right?

Reply:
yeah, since it's a hard coded string that we know is safe, there's no risk of injection, and so the bind variable seems unnecessary to me.